README.md

```diff
@@ -155,6 +155,17 @@ Hub for a list of all the supported tags.
 |-------------|----------------|
 | `/var/log`  |  Log storage   |
 
+## Ports ##
+
+The following ports are exposed by this container:
+
+| Port | Purpose        |
+|------|----------------|
+| 8080 | Example only; nothing is actually listening on the port |
+
+The sample [Docker composition](docker-compose.yml) publishes the
+exposed port at 8080.
```

Comment:
This "Ports" section should be updated for this particular Docker container.

Any similar sections ("Volumes", etc.) should be updated as well, if they haven't already.

Reply:
This appears to be a placeholder repo right now. Once a `first-commits` branch is PRed this information will change as part of that.

Reply:
@mcdonnnj - you are correct on both counts.

Reply:
Got it.  Thanks gents!